web_src/js/features/comp/ComboMarkdownEditor.js

```diff
@@ -14,19 +14,18 @@ let elementIdCounter = 0;
 
 /**
  * validate if the given textarea is non-empty.
- * @param {jQuery} $textarea
+ * @param {HTMLElement} textarea - The textarea element to be validated.
  * @returns {boolean} returns true if validation succeeded.
  */
-export function validateTextareaNonEmpty($textarea) {
+export function validateTextareaNonEmpty(textarea) {
   // When using EasyMDE, the original edit area HTML element is hidden, breaking HTML5 input validation.
   // The workaround (https://github.com/sparksuite/simplemde-markdown-editor/issues/324) doesn't work with contenteditable, so we just show an alert.
-  if (!$textarea.val()) {
-    if ($textarea.is(':visible')) {
-      $textarea.prop('required', true);
-      const $form = $textarea.parents('form');
-      $form[0]?.reportValidity();
+  if (!textarea.value) {
+    if (isVisible(textarea)) {
+      textarea.setAttribute('required', 'true');
+      const form = textarea.closest('form');
+      form?.reportValidity();
     } else {
-      // The alert won't hurt users too much, because we are dropping the EasyMDE and the check only occurs in a few places.
```

Comment:
This comment should be kept, because the code blow doesn't look that straightforward.

Reply:
Done https://github.com/go-gitea/gitea/pull/29271/commits/f658668951a7934f5e362b0b0d9526b67ecf17b2